cookbook/core/type_system/structured_dataset.py

```diff
@@ -87,18 +87,69 @@ def get_subset_df(
 ) -> Annotated[StructuredDataset, subset_cols]:
     df = df.open(pd.DataFrame).all()
     df = pd.concat([df, pd.DataFrame([[30]], columns=["Age"])])
-    # On specifying BigQuery uri for StructuredDataset, Flytekit will write pd.dataframe to a BigQuery table
+    # On specifying BigQuery uri for StructuredDataset, flytekit writes pd.dataframe to a BigQuery table
     return StructuredDataset(dataframe=df)
 
+# %%
+# Example of StructuredDataset With ``uri`` Argument
+# ==================================================
+#
+# As mentioned previously, if you specify BigQuery ``uri`` for StructuredDataset, flytekit writes the pandas dataframe to a BigQuery table.
+#
+# To write a dataframe to a BigQuery table, 
```

Comment:
```suggestion
# To write DataFrame to a BigQuery table, 
```

Reply:
This lead-in doesn't fit well with the bullet points thereafter. In the bullet points, you ask the user to set up GCP and the lead-in misleads the user into thinking that the bullet points are about creating a dataframe from a bigquery table.